### PR TITLE
test: Add edit tests for hostname (HMS-5184)

### PR DESCRIPTION
**Pull request by regexowl**
This adds an edit test for the Hostname step.<br/><br/>JIRA: [HMS-5184](https://issues.redhat.com/browse/HMS-5184)

### CODE REVIEW

File: src/test/fixtures/editMode.ts [hunk 1]
```diff
@@ -445,18 +451,27 @@ export const localeCreateBlueprintRequest: CreateBlueprintRequest = {
...
   id: mockBlueprintIds['locale'],
   description: mockBlueprintDescriptions['locale'],
 };
 
+export const hostnameCreateBlueprintRequest: CreateBlueprintRequest = {
+  ...baseCreateBlueprintRequest,
+  name: mockBlueprintNames['hostname'],
+  description: mockBlueprintDescriptions['hostname'],
+  customizations: {
+    hostname: 'baseimage',
```
Comment (mgold1234) [inline src/test/fixtures/editMode.ts:465]: wdyt maybe change the name to 
```suggestion
    hostname: 'baseImage',
```
or 
```suggestion
    hostname: 'base-image',
```

**Review by mgold1234 — COMMENTED** [review]
@regexowl /lgtm, small suggestion

File: src/test/fixtures/editMode.ts (hunk 1 above)
Comment (regexowl) [reply src/test/fixtures/editMode.ts]: Sure, the first suggestion might be invalid for a hostname, changed it to the second one.

**Review by mgold1234 — DISMISSED** [review]
/lgtm

**Pull request merged**